specs/src/networking.md

```diff
@@ -81,7 +87,10 @@ If a malicious block producer incorrectly computes the 2D Reed-Solomon code for
 Defined as `BadEncodingFraudProof`:
 
 ```protobuf
-{{#include ./proto/types.proto:BadEncodingFraudProof}}
+// BadEncodingFraudProof message definition (removed from specs)
+message BadEncodingFraudProof {
+  // See table below for field descriptions
+}
```

Comment:
We should prob remove BadEncodingFraudProof from the specs b/c AFAIK it was supported in celestia-node for a time but no longer is supported.